libs/designsystem/src/lib/scss/themes/_colors.scss

```diff
@@ -28,6 +28,8 @@ $text-colors: (
   'danger': #ee0d0d,
 );
 
+$focus-ring-color: rgb(77, 144, 254);
```

Comment:
not a kirby color? I expect you have your reasons so I have approved anyway :-)

Reply:
It IS a Kirby color added to the `_colors.scss` but it's utility color, not a brand color :)